dqnroute_internet_main.py

```diff
@@ -54,16 +65,19 @@ def main():
             os.remove(args.logfile)
         except FileNotFoundError:
             pass
-
-    actorSys = ActorSystem('multiprocQueueBase')
-    overlord = actorSys.createActor(RoutersOverlord, globalName='overlord')
-    actorSys.tell(overlord, RoutersOverlordInitMsg(graph=G,
-                                                   settings=run_params['settings'],
-                                                   results_file=args.results_file,
-                                                   logfile=args.logfile,
-                                                   router_type=args.router_type))
-    while not stop_it:
-        next(sys.stdin)
+    
+    env = simpy.Environment()
+    routers = {}
+    for node in G.nodes():
+        routers[node] = SimPyDumbRouter(env, node)
+    for node in G.nodes():
+        out_routers = [routers[v] for (_, v) in G.edges(node)]
+        routers[node].setNeighbours(out_routers)
+    env.process(sendRandomPkg(env, routers))
+    env.run(until=40)
+    
+    #while not stop_it:
+    #    next(sys.stdin)
```

Comment:
Is there any reason to comment out this piece of code instead of removing it altogether?